README.md

```diff
@@ -35,17 +35,17 @@ community to further enhance its capabilities and compatibility with different b
 ### Use cases
 
 The Outport Driver operates on a robust websocket connection architecture, supporting both server and client roles. The
-system is designed to seamlessly export data from the Multiversx blockchain node, which adopts a sharded architecture
+system is designed to seamlessly export data from the MultiversX blockchain node, which adopts a sharded architecture
 where each shard represents a separate running chain.
 
 To export data from a specific shard, you need to enable an observer node within that shard to export data. One can do
 that
 by enabling the **[HostDriverConfig].Enabled** flag to true
-[from this config file](https://github.com/multiversx/mx-chain-go/blob/master/cmd/node/config/external.toml)
+[from this config file](https://github.com/MultiversX/mx-chain-go/blob/master/cmd/node/config/external.toml)
 
-In our case of a network consisting of three shards, you would require three separate websocket receiver binaries, each
-responsible for receiving data from its respective shard. This distributed approach ensures data integrity and allows
-for shard-specific data processing.
+In our case of a network consisting of three shards and a metachain, you would require three separate websocket receiver
+binaries, each responsible for receiving data from its respective shard. This distributed approach ensures data
```

Comment:
now you need 4 websocket receiver binaries

Reply:
Right, missed it